### ISSUE TITLE
wp core install fails with Wordpress 4.4

### ISSUE THREAD

**Issue by hansfilipelo** [wp-cli/wp-cli#2273]
I'm running the following commands, in order:

```
wp core download --locale=sv_SE
wp core config --dbname=$userName --dbuser=$userName --dbpass=$userPassword --locale=sv_SE
wp core install --url=$FQDN --title='$userName website' --admin_user=$adminUser --admin_password=$wpAdminPassword --admin_email=$adminMail
```

This gives the following error. 

```
PHP Warning:  require(/path/to/my/wordpress/wp-includes/embed-functions.php): failed to open stream: No such file or directory in phar:///usr/bin/wp/php/utils-wp.php on line 62
Warning: require(/path/to/my/wordpress/wp-includes/embed-functions.php): failed to open stream: No such file or directory in phar:///usr/bin/wp/php/utils-wp.php on line 62
PHP Fatal error:  require(): Failed opening required '/path/to/my/wordpress/wp-includes/embed-functions.php' (include_path='phar:///usr/bin/wp/vendor/phpunit/phpunit-mock-objects:phar:///usr/bin/wp/vendor/phpunit/php-token-stream:phar:///usr/bin/wp/vendor/phpunit/php-code-coverage:phar:///usr/bin/wp/vendor/phpunit/phpunit:phar:///usr/bin/wp/vendor/symfony/yaml:.:/usr/share/php:/usr/share/pear') in phar:///usr/bin/wp/php/utils-wp.php on line 62
Fatal error: require(): Failed opening required '/path/to/my/wordpress/wp-includes/embed-functions.php' (include_path='phar:///usr/bin/wp/vendor/phpunit/phpunit-mock-objects:phar:///usr/bin/wp/vendor/phpunit/php-token-stream:phar:///usr/bin/wp/vendor/phpunit/php-code-coverage:phar:///usr/bin/wp/vendor/phpunit/phpunit:phar:///usr/bin/wp/vendor/symfony/yaml:.:/usr/share/php:/usr/share/pear') in phar:///usr/bin/wp/php/utils-wp.php on line 62
```

I'm running this on Debian 8 - using PHP 5.6.14-0+deb8u1.

**Comment by hansfilipelo** [comment]
Fixed with 0.21.1 - though this relase is not available as a .deb-file (yet?).

**Comment by danielbachhuber** [comment]
> though this relase is not available as a .deb-file (yet?).

Sorry about that. We'll get one up shortly https://github.com/wp-cli/builds/issues/29
